CCDB/src/ConditionsMQServer.cxx

```diff
@@ -48,19 +48,19 @@ void ConditionsMQServer::InitTask()
 {
   ParameterMQServer::InitTask();
   // Set first input
-  if (GetProperty(FirstInputType, "") == "OCDB") {
-    mCdbManager->setDefaultStorage(GetProperty(FirstInputName, "").c_str());
+  if (GetFirstInputType() == "OCDB") {
+    mCdbManager->setDefaultStorage(GetFirstInputName().c_str());
   }
 
   // Set second input
-  if (GetProperty(SecondInputType, "") == "OCDB") {
-    mCdbManager->setDefaultStorage(GetProperty(SecondInputName, "").c_str());
+  if (GetSecondInputType() == "OCDB") {
+    mCdbManager->setDefaultStorage(GetSecondInputName().c_str());
   }
 
   // Set output
-  if (GetProperty(OutputName, "") != "") {
-    if (GetProperty(OutputType, "") == "OCDB") {
-      mCdbManager->setDefaultStorage(GetProperty(OutputName, "").c_str());
+  if (GetOutputName() != "") {
+    if (GetOutputType() == "OCDB") {
+      mCdbManager->setDefaultStorage(GetOutputName().c_str());
```

Comment:
The two inputs are given to `FairRuntimeDb`.
When both inputs are provided, and a parameter exists in both, first input will be taken.
There are more details here: https://fairroot.gsi.de/?q=node/28.
But I will also extend the doxygen of the ParameterMQServer.